<launch>
  <arg name="port" default="/dev/ttyUSB0" />
  <arg name="sys" default="1" />
  <arg name="enc" default="0" />
  <arg name="io" default="0" />
  <arg name="velocity" default="1" />
  <arg name="rate" default="50" />
  <arg name="enc_pulse" default="512" />
  <arg name="pulse_rev_left" default="4096" />
  <arg name="pulse_rev_right" default="4096" />
  <arg name="gear_ratio" default="30" />
  <arg name="sprocket" default="83" />
  <arg name="track" default="0.6103" />
  <arg name="cmd_vel" default="1" />

  <node pkg="robo_base" type="robo_base" name="robo_base" output="screen"> 
    <param name="port" value="$(arg port)" />
    <param name="sys" value="$(arg sys)" />
    <param name="enc" value="$(arg enc)" />
    <param name="io" value="$(arg io)" />
    <param name="velocity" value="$(arg velocity)" />
    <param name="rate" value="$(arg rate)" />
    <param name="enc_pulse" value="$(arg enc_pulse)" />
    <param name="pulse_rev_left" value="$(arg pulse_rev_left)" />
    <param name="pulse_rev_right" value="$(arg pulse_rev_right)" />
    <param name="sprocket" value="$(arg sprocket)" />
    <param name="track" value="$(arg track)" />
    <param name="cmd_vel" value="$(arg cmd_vel)" />
  </node>
</launch>
